index.html

```diff
@@ -451,11 +451,13 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         <b class="header">each</b><code>_.each(list, iteratee, [context])</code>
         <span class="alias">Alias: <b>forEach</b></span>
         <br />
-        Iterates over a <b>list</b> of elements, yielding each in turn to an <b>iteratee</b>
-        function. The <b>iteratee</b> is bound to the <b>context</b> object, if one is
+        Iterates over a <b>list</b> of elements, yielding each in turn to an
+        <a href="#iteratee"><b>iteratee</b></a> function.
```

Comment:
Any good suggestion here? I can't come up with anything better than `callback` instead of `iteratee` . This was previously rejected in #1767 and I'm not sure this deserve much bikeshedding.